proto/signalfx_metrics.proto

```diff
@@ -17,22 +17,20 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.goproto_registration) = true;
 
 enum MetricType {
-  /**
-   * Numerical: Periodic, instantaneous measurement of some state.
-   */
+  // Periodic, instantaneous numerical measurement of some state.
   GAUGE = 0;
-  /**
-   * Numerical: Count of occurrences. Generally non-negative integers.
-   */
+
+  // Counts of occurrences since last report. Generally non-negative integers.
   COUNTER = 1;
-  /**
-   * String: Used for non-continuous quantities (that is, measurements where there is a fixed
-   * set of meaningful values). This is essentially a special case of gauge.
-   */
+
+  // Used for non-continuous quantities (that is, measurements where there is a fixed
+  // set of meaningful values). This is essentially a special case of gauge that reports a string value.
```

Comment:
i don't even know what will happen if an enum is sent in... i don't think this is actually implemented.

Reply:
No idea, I've tried to document it, if you think I should remove it I am happy to do it.